modules/packages/Futures.chpl

```diff
@@ -18,11 +18,9 @@
  * limitations under the License.
  */
 
-/*
-
-.. |---| unicode:: U+2014
+/* Containers for accessing the results of asynchronous execution.
 
-Containers for accessing the results of asynchronous execution.
+.. |---| unicode:: U+2014   TODO: What is this?
```

Comment:
What is this?

Reply:
u+2014 is an m-dash, FWIW

Reply:
Ah, now I see.  This is defining a macro-style replacement for `---` for use within the documentation itself.  That's more advanced than my rst/sphinx knew about.